src/scrabble.js

```diff
@@ -1,5 +1,59 @@
-function scrabble() {
+const array1 = ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T']
+const array2 = ['D', 'G']
+const array3 = ['B', 'C', 'M', 'P']
+const array4 = ['F', 'H', 'V', 'W', 'Y']
+const array5 = ['K']
+const array8 = ['J', 'X']
+const array10 = ['Q', 'Z']
 
+function letterValue(letter) {
+  let value = 0
+  const letterUpper = letter.toUpperCase()
+  // console.log(letterUpper)
+  if (array1.includes(letterUpper)) {
```

Comment:
using array.includes vs what id did of applying if statements to each character in the string im a big fan of